src/frontend/screens/Settings/sections/LogSettings/index.tsx

```diff
@@ -115,37 +121,58 @@ export default function LogSettings() {
           'Join our Discord and look for the "#-support" section. Read the pinned "Read Me First | Frequently Asked Questions" thread and follow the instructions to share these logs and any relevant information about your problem.'
         )}
       </p>
-      {isDefault && (
+      <div className="logs-wrapper">
         <span className="log-buttongroup">
-          <a
-            className={`log-buttons ${!defaultLast ? 'log-choosen' : ''}`}
-            onClick={() => {
-              setRefreshing(true)
-              setDefaultLast(false)
-            }}
-            title={t('setting.log.current-log')}
-          >
-            {t('setting.log.current-log', 'Current log')}
-          </a>
-          <a
-            className={`log-buttons ${defaultLast ? 'log-choosen' : ''}`}
-            onClick={() => {
-              setRefreshing(true)
-              setDefaultLast(true)
-            }}
-            title={t('setting.log.last-log')}
-          >
-            {t('setting.log.last-log', 'Last Log')}
-          </a>
-        </span>
-      )}
-      {refreshing ? (
-        <span className="log-box">
-          <UpdateComponent inline />
+          {[
+            ['Heroic', 'heroic'],
+            ['Epic/Legendary', 'legendary'],
+            ['GOG', 'gogdl'],
+            ['Amazon/Nile', 'nile']
```

Comment:
I didn't add translations for these names on purpose, I've seen them translated and translated incorrectly when they shouldn't be translated actually